### ISSUE TITLE
HTTP 429 error on initial boot

### ISSUE THREAD

**Issue by albill** [urbit/urbit#4917]
I've had a comet running but just got a planet. I'm resetting up my instance to use the planet and getting an HTTP 429 error on first boot with my planet key.

Output is below:

`
$ ./urbit -p 49152 -w poslet-montyl -k ./poslet-montyl-1.key
~
urbit 1.5
boot: home is /home/albill/urbit/poslet-montyl
loom: mapped 2048MB
lite: arvo formula 79925cca
lite: core 59f6958
lite: final state 59f6958
boot: downloading pill https://bootstrap.urbit.org/urbit-v1.5.pill
boot: retrieving latest block
error fetching http://eth-mainnet.urbit.org:8545: HTTP 429
`

**Comment by philipcmonk** [comment]
Yeah we're running into Ethereum node rate limits, @Fang- is looking into it
